package/upgrade/upgrade_manifests.sh

```diff
@@ -281,6 +281,9 @@ upgrade_rancher() {
   wait_rollout cattle-system deployment rancher-webhook
   echo "Wait for cluster settling down..."
   wait_capi_cluster fleet-local local $pre_generation
+  patch_fleet_cluster
+  sleep 15 #wait a few seconds to ensure that a new deployment is created. Without this wait_rollout fails as deployment is never found
```

Comment:
Could we have something more robust to detect the existence of `fleet-agent` Deployment? I'm afraid that under heavy load, the rancher manager might have a chance of not responding in 15 seconds. Thanks!